scripts/docker/alpine/3.10/release.Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM alpine:3.10
+
+RUN apk update \
+    && apk add --no-cache --upgrade\
+        bison \
+        build-base \
+        cmake \
+        curl \
+        diffutils \
```

Comment:
Why is this needed? libgit2 would be needed for two plugins.

Reply:
I blindly copied the other alpine image and adapted it quickly. :see_no_evil:

Reply:
Please do note that this is only for building. Then we copy it to a fresh image with only bash installed and the layers are squashed so the image should have minimal size.

Basically my only goal was to provide a minimal image and drop the user into a shell where kdb is available. I do agree that it would be nice to add more plugins for a rich Elektra experience.

Reply:
Yes, this can also be done in the next release. I hope everything for the tutorial is included in the current image?

Reply:
I did not go through the tutorial. I'll update the image if something is missing. (hopefully alpine has packages for everything needed)